seqr/views/utils/dataset_utils.py

```diff
@@ -366,15 +422,17 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
     header = _parse_tsv_row(next(f))
     missing_cols = set(expected_columns) - set(header)
     if missing_cols:
-        raise ValueError(f'Invalid file: missing column(s) {", ".join(sorted(missing_cols))}')
+        raise ValueError(f'Invalid file: missing column(s): {", ".join(sorted(missing_cols))}')
 
     sample_id_to_tissue_type = {}
     samples_with_conflict_tissues = defaultdict(set)
+    miss_matches = []
     for line in tqdm(f, unit=' rows'):
         row = dict(zip(header, _parse_tsv_row(line)))
         for sample_id, row_dict in parse_row(row):
             tissue_type = TISSUE_TYPE_MAP.get(row_dict.pop(TISSUE_COL, None))
             project = row_dict.pop(PROJECT_COL)
+            indiv_id = row_dict.pop(INDIV_ID_COL, None)
```

Comment:
Because we need to pop the `individual id` field before miss-match detection and error message generation, otherwise, the row_dict could have an `individual_id` field, but the existing_data doesn't.

Reply:
I updated the test to catch treating the identical rows as miss-matches.